src/qt/sendcoinsdialog.cpp

```diff
@@ -822,6 +822,12 @@ void SendCoinsDialog::coinControlUpdateLabels()
     // set pay amounts
     CoinControlDialog::payAmounts.clear();
     CoinControlDialog::fSubtractFeeFromAmount = false;
+    if (ui->radioSmartFee->isChecked()) {
+        CoinControlDialog::coinControl->nConfirmTarget = ui->sliderSmartFee->maximum() - ui->sliderSmartFee->value() + 2;
```

Comment:
Seems strange that all these CoinControlDialog members are static, but of course the pr isn't changing that. This seems like a clean fix given the current organization.